### PR TITLE
Update Italian language

### PR DESCRIPTION
**Pull request by bovirus**
@selvanair 

Restore a lienr emoved (but necessary) and presnt in english file).

### CODE REVIEW

File: res/openvpn-gui-res-it.rc [hunk 1]
```diff
@@ -411,7 +411,8 @@ Per maggiori dettagli consulta il file log (%ls) ."
     IDS_NFO_SERVICE_STOPPED "Servizio OpenVPN arrestato."
     IDS_NFO_ACTIVE_CONN_EXIT "Ci sono ancora connessioni attive che verranno chiuse se esci dall'interfaccia di OpenVPN.\
 \n\nSei sicuro di voler uscire?"
-    IDS_ERR_PARSE_MGMT_OPTION "Impossibile analizzare l'opzione --management in <%ls%ls>."
+    IDS_ERR_PARSE_MGMT_OPTION "Impossibile analizzare l'opzione --management in <%ls%ls>. \
+L'allegato alla connessione automatica richiede l'opzione --management nel file config."
```
Comment (ordex) [inline res/openvpn-gui-res-it.rc:415]: Hi, was this the same sentence that was used before?
The English text says "Attaching to auto-started connections require --management option in the config file."
There is truly no "allegato" in here, please.

**Review by ordex — CHANGES_REQUESTED** [review]
the new sentence is not appropriate

File: res/openvpn-gui-res-it.rc (hunk 1 above)
Comment (bovirus) [reply res/openvpn-gui-res-it.rc:415]: Why is available in english language?

Comment (bovirus) [reply res/openvpn-gui-res-it.rc:415]: "Could be ok "Il collegamento a connessioni avviate automaticamente richiede l'opzione --management nel file di configurazione."?

Comment (bovirus) [reply res/openvpn-gui-res-it.rc]: Ok. Fixed. Thanks.

Comment (ordex) [reply res/openvpn-gui-res-it.rc]: please squash everything in one commit

**Comment by ordex** [comment]
Why did you close the PR? You could just squash the two commits and force-push to the same branch.
The PR would then be updated accordingly

**Comment by bovirus** [comment]
@ordex 
I prefer to make the changes in one step.
Clear and to avoid mnisudnerstanding.
What's the problem to use this system?

**Comment by bovirus** [comment]
@ordex 

The previous comment was go over because I applied the string that we and you are agree.
Sorry but I didn'ts ee the problem.
Thanks.

**Comment by schwabe** [comment]
@bovirus your behaviour is not what you are expected to do and who pull request are supposed to be used. By working this way you basically annoy everything but not doing things the correct way. E.g. I now get multiple notifcation of new pull request being opened and closed, which is annoying. Just we really would like to work and use Github like everyone else.

**Comment by schwabe** [comment]
@ordex already told you. SQUASH the commit and update the EXISTING Pr. NOT opening a NEW Pr. Thanks

**Comment by bovirus** [comment]
@schwabe

I'm afraid but I'm not able to do it.
I don't ahve the right competencies.

**Comment by selvanair** [comment]
@bovirus I can squash on merge (rebase and merge option in github) if you are okay with it. You have to just reopen the PR.

**Comment by bovirus** [comment]
@selvanair 
I beleiev that I cannot re-open PR.
To create the new PR I deleted the previous fork.
Pelase tell me how can help you.
Thanks.

**Pull request closed without merging**